src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsEventSourceLifetime.cs

```diff
@@ -21,6 +21,7 @@ public class TestsEventSourceLifetime
         /// </summary>
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // non-Windows EventSources don't have lifetime
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "ExerciseEventSource use private member reflection")]
```

Comment:
Thanks @morganbr.  I think the right fix for this test is to use InternalsVisibleTo.  The goal of the test is to make sure that the EventSource gets properly disposed.  It's fine to remove the reflection and just use InternalsVisibleTo for the test.  @tijoytom can you make this change instead of disabling the test?